tests/src/sanity.rs

```diff
@@ -286,6 +290,7 @@ mod tests {
         // docker build -t test . -f mirrord-agent/Dockerfile
         // minikube load image test:latest
         let mut env = HashMap::new();
+        env.insert("COMPlus_EnableDiagnostics", "0");
         env.insert("MIRRORD_AGENT_IMAGE", "test");
         env.insert("MIRRORD_CHECK_VERSION", "false");
         env.insert("MIRRORD_AGENT_RUST_LOG", "warn,mirrord=debug");
```

Comment:
can you please change all log levels to trace, might provide more context what is going wrong

Reply:
Thank you!

[Here](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-6.0#set-log-level-by-command-line-environment-variables-and-other-configuration) is dotnet specific instructions for setting the log level as well.

Reply:
nice, but for now can you change it back to 0 so that we can just focus on file-based logs? thanks!

Reply:
# 😅
Sorry! I got carried away with enabling the tracing. I'll fix this